sample/.editorconfig

```diff
@@ -13,3 +13,4 @@ dotnet_diagnostic.CA1848.severity = none
 dotnet_diagnostic.S2629.severity = none
 dotnet_diagnostic.S2696.severity = none
 dotnet_diagnostic.CA2254.severity = none
+dotnet_diagnostic.RE0001.severity = none
```

Comment:
I guess this can be removed!?
Don't see where you added a Regex.

Reply:
I didn't add any regex. Had to add that to keep Visual Studio from complaining about the regex on the Cron parameters. It's from the change you made adding the following: `[StringSyntax(StringSyntaxAttribute.Regex)] string cronExpression`

Was that unintentional?

Reply:
Oh - yes. Not sure why I thought that should help with regex -.-

Sorry can you please remove the `[StringSyntax(StringSyntaxAttribute.Regex)]` part.

Reply:
done